UNRELEASED.md

```diff
@@ -2,6 +2,11 @@
 
 > Place your changes below this line.
 
+**Breaking:**
```

Comment:
Not sure if this makes it breaking or added...

Reply:
It will be added as it didn't exist previously 😄

Reply:
But it's required, which means it'll break badge theming for any existing consumers of it. 

They'll need manually add this theme prop for their code to keep working, which makes this breaking. It's ok to do breaking changes, especially as we only have one consumer of this library and we can manage the upgrade for them.

Reply:
Unrelated but I think we should relax to whole thing to warning instead of errors (not defining all theme props)

Reply:
It's a prop type warning so it won't actually cause any errors in prod, only dev. But it will disable theming in components that don't provide all the necessary attributes, which I still think is important because we do generally want to ensure that all required props are provided right? Otherwise we run the risk of people seeing partially-themed components which will look bad.

Reply:
Yeah if not all theme props are provided for a component it will not allow partial theming - it was a change we did a few months ago to prevent what Shaun said about partial themed components

Reply:
Right, yeah having partner branding in mind this makes sense